src/api/app/views/webui2/webui/patchinfo/form/_issues.html.haml

```diff
@@ -0,0 +1,31 @@
+.form-group
+  = form.label(:issues, class: 'd-block')
+
+  %ul#issues
+    - issues.each do |issue|
```

Comment:
When submitting an invalid patchinfo I get an `undefined method `each' for nil:NilClass` caused by this line.

Reply:
Fixed